core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -926,6 +926,14 @@ public String toString() {
   /**
    * UFS related properties.
    */
+  public static final PropertyKey UNDERFS_STRICT_VERSION_MATCH_ENABLED =
+      Builder.booleanBuilder(Name.UNDERFS_STRICT_VERSION_MATCH_ENABLED)
+          .setDefaultValue(false)
+          .setDescription("Strict compare ufs version.")
```

Comment:
```suggestion
          .setDescription("When enabled, Alluxio finds the UFS connector by strict version matching. Otherwise only version prefix is compared.")
```